spec/workers/publish_retirement_event_worker_spec.rb

```diff
@@ -25,5 +25,11 @@
         .with("workflow_counters", payload_expectation)
       worker.perform(workflow.id)
     end
+
+    it "should publish via EventStream" do
```

Comment:
nitpicky - maybe change the spec name to match the expectation, i.e. publish via kinesis.